jujugui/static/gui/src/app/app.js

```diff
@@ -1260,16 +1260,13 @@ YUI.add('juju-gui', function(Y) {
         if (!auth) {
           return;
         }
-        this.state.changeState({
-          model: null,
-          profile: auth.rootUserName,
-          root: null,
-          store: null
-        });
+        views.utils.showProfile(
```

Comment:
Ah, good that we reuse this existing method. Maybe we should set the store field to null as well in utils._showProfile?